### DISCUSSION TITLE
Ignoring a request / do not send response

### DISCUSSION THREAD

**Discussion by airwillow** [mockoon/mockoon#1102] [Help]
Is it possible to ignore a request and configure mockoon to not send any response.

I would like to simulate a network failure or an API failure. Of course, switch off mockoon is not the right answer ;) 

I give you a simple use case : 

I must call, in a batch, 100 times the API to get some users info from user1 to user100
I need to simulate a failure when requesting user58 to check if my batch do what it is supposed to do in this case 

So i would like to set a condition on this route (or in a param of the request) to tell mockoon to not reply anything.

**Comment by 255kb** [comment]
There is no such thing as not responding, but you can try two setups with two responses (e.g. 5xx and 2xx): 

- enable the "random responses" mode: 
   ![image](https://github.com/mockoon/mockoon/assets/7489814/50c3c040-483d-4d73-aa1a-dc8f8294a10e)

- set the 200 response as default (blue flag), use a "Request number" response rule to on the 500 route to serve this response at request number 58:  
  ![image](https://github.com/mockoon/mockoon/assets/7489814/f39d92f2-6535-4c79-8bfe-d7b3be021c53)
  ![image](https://github.com/mockoon/mockoon/assets/7489814/3666aed1-557d-462f-9fcc-67a8909e497a)

**Reply by airwillow** [reply]
Thanks for your answer, it's interesting :) . But it doesn't suit my use case unfortunately.

I want to check if my app reacts correctly when an API is unavailable and that is what is would like to mock.

But I will keep your solution for mocking others use cases :)
